site/assets/scss/_component-examples.scss

```diff
@@ -114,11 +115,12 @@
 //
 
 .bd-example-row [class^="col"],
-.bd-example-cssgrid .grid > * {
+.bd-example-cols [class^="col"] > *,
+.bd-example-cssgrid [class*="grid"] > * {
```

Comment:
**[BS]** Should we introduce such rule for the [grid example](https://deploy-preview-1757--boosted.netlify.app/docs/5.2/examples/grid/) to be more consistent ?